pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -22,6 +24,28 @@
     _strip_large_initializer_raw_data
 
 
+def _model_to_graph_with_value_names(*args, add_value_names=True, **kwargs):
+    g, p, o = torch_model_to_graph(*args, **kwargs)
+    if not add_value_names:
+        return g, p, o
+
+    for n in g.nodes():
+        for v in itertools.chain(n.inputs(), n.outputs()):
+            if not v.debugName().isnumeric():
+                continue
+            old_name = v.debugName()
+            new_name = 'v{}_{}'.format(old_name, n.kind().split('::')[-1])
+            v.setDebugName(new_name)
+            if old_name in p:
+                i = p[old_name]
+                del p[old_name]
+                p[new_name] = i
+    return g, p, o
+
+
+torch.onnx.utils._model_to_graph = _model_to_graph_with_value_names
```

Comment:
memo: replacing only in ppe.onnx.export would be more cleaner (thank you @kmaehashi san)